keyboards/handwired/stef9998/split_5x7/keymaps/default/keymap.c

```diff
@@ -0,0 +1,94 @@
+/*
+Copyright 2022 stef9998
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#include QMK_KEYBOARD_H
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// You can also skip layer-names entirely and just use numbers.
+#define _QWERTY 0
+#define _FN 3
+#define _SYM 4
+#define _CFG 7//config
```

Comment:
Skipping numbers like this introduces empty layers and artificially increases the firmware size.

Enums are easier to manage, as it automatically increment the numbers. 

```suggestion
enum custom_layer {
    _QWERTY,
    _FN,
    _SYM,
    _CFG,
};
```